__tests__/utils/token.test.js

```diff
@@ -0,0 +1,53 @@
+import {
+  setAccessToken,
+  getAccessToken,
+  setRefreshToken,
+  getRefreshToken,
+  getUserId,
+  setUserId,
+  setToken,
+  getToken,
+  setAnonymousToken,
+  getAnonymousToken,
+  removeLocalStorege,
+  removeAccessToken,
+} from '../../src/utils/token';
+describe('testing function related to the token', () => {
+  it('Test Set Token & Get Token', async () => {
+    await setAccessToken('testToken');
+    expect(await getAccessToken()).toEqual('testToken');
+  });
+  it('Test Refresh Token', async () => {
+    await setRefreshToken({id: '1234'});
+    expect(await getRefreshToken()).toEqual('{"id":"1234"}');
+  });
+
+  it('test setToken should run correctly ', async () => {
+    await setToken('testToken');
+    expect(await getToken()).toEqual('testToken');
+  });
+
+  it('setAnonymousToken should run correctly', async () => {
+    await setAnonymousToken('tokenAnonym');
+    expect(await getAnonymousToken()).toEqual('tokenAnonym');
+  });
+
+  it('setUserUd should run correctly', async () => {
+    await setAnonymousToken('tokenAnonym');
+    expect(await getAnonymousToken()).toEqual('tokenAnonym');
+  });
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

There seems to be a copy-paste error in this test case. The test description says it's testing `setUserId` but the implementation is for `setAnonymousToken`. Please correct the function being tested and its corresponding assertion.
```diff
-  it('setUserUd should run correctly', async () => {
-    await setAnonymousToken('tokenAnonym');
-    expect(await getAnonymousToken()).toEqual('tokenAnonym');
-  });
+  it('setUserId should run correctly', async () => {
+    const userId = 'testUserId';
+    await setUserId(userId);
+    expect(await getUserId()).toEqual(userId);
+  });
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->